docs/CommunityProjects.md

```diff
@@ -15,6 +15,7 @@ Name | Owner | Category | Description | License
 [Easy UI5](https://github.com/SAP/generator-easy-ui5) | [@IObert](https://github.com/IObert) | Tool | Generator for OpenUI5 apps using the official UI5 tooling and supporting deployment to SAP Cloud Platform. | Apache 2.0
 [Express UI5](https://github.com/handy-man/Expressui5) | [@handy-man](https://github.com/handy-man) | Tool | CLI tool to create ui5 projects quickly from developer maintained templates. | Apache 2.0
 [grunt-nwabap-ui5uploader](https://github.com/pfefferf/grunt-nwabap-ui5uploader) | [@pfefferf](https://github.com/pfefferf) | Tool | A Grunt plugin to upload UI5 sources to a SAP NetWeaver ABAP system. | Apache 2.0
+[UI5-JSX](https://github.com/KrazyJO/ui5-jsx) | [@KrazyJO](https://github.com/KrazyJO) | Tool | JSX to UI Babel Plugin extended for Render API 2. | Apache 2.0
```

Comment:
Hi,

Thank you for the contribution.

Is it fine for you to change from [UI5-JSX](https://github.com/KrazyJO/ui5-jsx) to [JSX to UI5](https://github.com/KrazyJO/ui5-jsx), that the content on the page will be alphabetical sorted?